[ES] Add `Changelog` (#8740)
ppy/osu-wiki: wiki/Changelog/es.md

---
tags:
  - history
  - historia
---

# Registro de cambios

> <!-- for the purpose of learning the history through the Wayback Machine (https://web.archive.org/), the changelog is available at different URLs:
>   - http://osu.ppy.sh/?p=changelog
>   - http://osu.ppy.sh/p/changelog
> --->
> 

Reviewer: ```suggestion
```

> El **[registro de cambios](https://osu.ppy.sh/home/changelog)** es una página donde él [equipo de desarrollo de osu!](/wiki/People/The_Team/Developers) detallan sus mejoras, adiciones, y correcciones de errores del día a día al juego. Todos los cambios de versiones anteriores son registrados aquí con fin de ser archivado.

Reviewer: ```suggestion
El **[registro de cambios](https://osu.ppy.sh/home/changelog)** es una página donde el [equipo de desarrollo de osu!](/wiki/People/The_Team/Developers) detallan sus mejoras, adiciones y correcciones de errores día a día al juego. Todos los cambios de versiones anteriores se registran aquí con fines de archivo.
```

> Los registros de cambios para el sitio web y osu!(lazer)son generados automáticamente basado en versiones de GitHub de esos proyectos, mientras que el resto se mantienen manualmente.

Reviewer: ```suggestion
Los registros de cambios para el sitio web y osu!(lazer) se generan automáticamente en función de las versiones de GitHub de esos proyectos, mientras que el resto se mantiene manualmente.
```

## Historia

> <!-- TODO(TicClick): wrap the images in infoboxes -->
> 

Reviewer: ```suggestion
```

![](img/changelog-web-2007.png "El registro de cambios web en 2007")

> El registro de cambios fue creado por peppy el 11 de septiembre de 2007 en un hilo del foro llamado "[Official Development Changelog](https://osu.ppy.sh/community/forums/topics/15)", donde él listaba cambios y correcciones de errores, y ocasionalmente compartía información sobre planes futuros.

Reviewer: ```suggestion
El registro de cambios fue creado por peppy el 11 de septiembre de 2007 en un hilo del foro dedicado titulado «[Official Development Changelog](https://osu.ppy.sh/community/forums/topics/15)», donde enumeraba cambios notables y correcciones de errores y, a veces, compartía información sobre próximos planes.
```

> ![](img/changelog-osume-2013.png "Registro de cambios en el osu! updater (2013)")

Reviewer: ```suggestion
![](img/changelog-osume-2013.png "Registro de cambios en el actualizador de osu! (2013)")
```

> En octubre, 2007, una versión web del registro de cambio [se hizo disponible](https://osu.ppy.sh/community/forums/posts/2499) tanto a través del sitio web y el osu! updater.

Reviewer: ```suggestion
En octubre de 2007, una versión web del registro de cambios [estuvo disponible](https://osu.ppy.sh/community/forums/posts/2499) tanto a través del sitio web como del actualizador de osu!.
```

> En octubre 25, 2009, el registro de cambios [se hizo disponible vía fuente RSS](https://osu.ppy.sh/community/forums/topics/19137).

Reviewer: ```suggestion
El 25 de octubre de 2009, el registro de cambios [estuvo disponible a través de una fuente RSS](https://osu.ppy.sh/community/forums/topics/19137).
```

![](img/build-propagation-graph-2015.png "Distribución de versiones de osu! (2015)")

> En octubre 28, 2015, una gráfica con la distribución de versiones de osu! a través de la base de jugadores [se añadió al registro de cambios](https://web.archive.org/web/20151103161516/http://osu.ppy.sh:80/p/changelog), haciendo las estadísticas abiertas a todo público. Al mismo tiempo, se volvió posible filtrar el registro de cambios por canal de actualizaciones.

Reviewer: ```suggestion
El 28 de octubre de 2015, [se agregó al registro de cambios](https://web.archive.org/web/20151103161516/http://osu.ppy.sh:80/p/changelog) un gráfico con la distribución de versiones de osu! en la base de jugadores, lo que hizo que las estadísticas estuvieran abiertas al público en general. Al mismo tiempo, se hizo posible filtrar el registro de cambios por canales de actualizaciones.
```

> ## Ver también

Reviewer: ```suggestion
## Véase también
```

> Además de los cambios listados en el sitio web, [el blog de peppy](https://blog.ppy.sh/) puede considerarse una forma de registro de cambio. A lo largo de los años, ha acumulado una amplia colección de publicaciones sobre osu!, su desarrollo, y ecosistema, incluyendo próximos planes y notas de las reuniones de los desarrolladores.

Reviewer: ```suggestion
Además de los cambios enumerados en el sitio web, el [blog de peppy](https://blog.ppy.sh/) también puede considerarse una forma de registro de cambios. A lo largo de los años, ha acumulado una gran colección de publicaciones centradas en osu!, su desarrollo y su ecosistema, incluidos los próximos planes y notas de reuniones de desarrolladores.
```
